packages/changed-elements-react/src/widgets/ChangedElementsWidget.tsx

```diff
@@ -302,7 +302,7 @@ export class ChangedElementsWidget extends Component<ChangedElementsWidgetProps,
         </IconButton>
         {
           this.props.useV2Widget &&
-          <InfoButton title={IModelApp.localization.getLocalizedString("VersionCompare:versionCompare.versionCompare")} message={this._widgetInfo} />
+          <InfoButton dataTestId="⁠comparison-legend-widget-info" title={IModelApp.localization.getLocalizedString("VersionCompare:versionCompare.versionCompare")} message={this._widgetInfo} />
```

Comment:
Why do buttons need data-testid? Can't someone just match the button by role=button and the button label?

Reply:
Oh, these buttons don't have user-readable labels. That's fine then.